libs/community/langchain_community/llms/databricks.py

```diff
@@ -225,7 +224,9 @@ def _is_hex_string(data: str) -> bool:
 def _load_pickled_fn_from_hex_string(data: str) -> Callable:
     """Loads a pickled function from a hexadecimal string."""
     try:
-        return pickle.loads(bytes.fromhex(data))
+        import cloudpickle
+
+        return cloudpickle.loads(bytes.fromhex(data))
     except Exception as e:
```

Comment:
may want to separately catch import errors so you can provide a helpful error message (e.g. "please install cloudpickle with ...")